app/controllers/posts_controller.rb

```diff
@@ -1,23 +1,32 @@
 class PostsController < ApplicationController
-  def new
-    @post = Post.new
-    # @user = User.new
-    # @current_user = @user.current_user
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts.includes(:comments)
   end
 
-  def create
-    @post = Post.new(post_params)
-    @post.author = current_user
-    if @post.save
-      redirect_to
-    else
-      render :new
-    end
+  def show
+    @comments = Post.find(params[:id]).comments
   end
 
-  private
+  # def new
+  #   @post = Post.new
+  #   # @user = User.new
+  #   # @current_user = @user.current_user
+  # end
 
-  def post_params
-    params.require(:post).permit(:title, :text)
-  end
+  # def create
+  #   @post = Post.new(post_params)
+  #   @post.author = current_user
+  #   if @post.save
+  #     redirect_to
+  #   else
+  #     render :new
+  #   end
+  # end
+
+  # private
+
+  # def post_params
+  #   params.require(:post).permit(:title, :text)
+  # end
```

Comment:
- OPTIONAL:  **I strongly recommend removing commented code. Please make the code clean**